spec/mailers/user_mailer_spec.rb

```diff
@@ -659,11 +661,171 @@ def expect_email_body_to_have_help_and_contact_links
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.state_id.info_eipp'),
+            )
+          end
+        end
+
+        it 'renders Questions? and Learn more link only once' do
+          expect(mail.html_part.body).to have_content(
+            t('in_person_proofing.body.barcode.questions'),
+          ).once
+          expect(mail.html_part.body).to have_link(
+            t('in_person_proofing.body.barcode.learn_more'),
+            href: MarketingSite.help_center_article_url(
+              category: 'verify-your-identity',
+              article: 'verify-your-identity-in-person',
+            ),
+          ).once
+        end
+
+        context 'template displays additional Enhanced In-Person Proofing specific content' do
+          it 'renders What to bring section' do
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.headings.barcode_what_to_bring'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.body.barcode.what_to_bring'),
+            )
+          end
+
+          it 'renders Option 1 content' do
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_bring_id.heading'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_bring_id.info'),
+            )
+          end
+
+          it 'renders Option 2 content' do
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_what_to_bring.heading'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_what_to_bring.info'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_passport.heading'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_passport.info'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_military_id.heading'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_military_id.info'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.heading'),
+            )
+            expect(mail.html_part.body).to have_content(
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.info'),
+            )
+            t('in_person_proofing.process.eipp_state_id_supporting_docs.info_list').each do |item|
+              expect(mail.html_part.body).to have_content(strip_tags(item))
+            end
```

Comment:
could maybe simplify this spec (and others in this file) with an array?
```suggestion
            aggregate_failures do
              [
                t('in_person_proofing.process.eipp_what_to_bring.heading'),
                t('in_person_proofing.process.eipp_what_to_bring.info'),
                t('in_person_proofing.process.eipp_state_id_passport.heading'),
                t('in_person_proofing.process.eipp_state_id_passport.info'),
                t('in_person_proofing.process.eipp_state_id_military_id.heading'),
                t('in_person_proofing.process.eipp_state_id_military_id.info'),
                t('in_person_proofing.process.eipp_state_id_supporting_docs.heading'),
                t('in_person_proofing.process.eipp_state_id_supporting_docs.info'),
                t('in_person_proofing.process.eipp_state_id_supporting_docs.info_list'),
              ].each do |copy|
                Array(copy).each do |part|
                  expect(mail.html_part.body).to contain(part)
                end
              end
```

Reply:
Good idea- this file is getting long. I cleaned up specs in this file and another file that had similar set up.
Resolved with commit 255a292c163f9bf91fed595b3ec1d65367f12680